spec/support/global_helpers.rb

```diff
@@ -0,0 +1,11 @@
+module GlobalHelpers
```

Comment:
I'd be inclined to call this `EnvHelpers` or something like that, to make its purpose more clear.